incubator/rethinkdb-cluster/extra/create-gce-storageclass.yaml

```diff
@@ -0,0 +1,7 @@
+kind: StorageClass
```

Comment:
Not sure if it is necessary to supply this manifest. It is fairly standard stuff.  At least, I would mention its purpose in the readme file.